src/content/en/updates/2019/02/using-twa.md

```diff
@@ -442,6 +442,119 @@ adb logcat | grep -e OriginVerifier -e digital_asset_links
 With the upload APK generated, you can now [upload the app to the Play
 Store](https://developer.android.com/studio/publish/upload-bundle).
 
+## Adding a Splash Screen {: #making-a-splash }
+
+Starting on **Chrome 75**, Trusted Web Activities are have support for Splash Screens.
+The Splash Screen can be added by adding a few new image files and configurations to the
+project.
+
+Make sure to update to **Chrome 75 or above** and use the
+[latest version of TWA Support Library](https://jitpack.io/#GoogleChrome/custom-tabs-client).
+
+### Generating the images for the Splash Screen {: #create-images }
+Android devices can have different [screen sizes](https://developer.android.com/training/multiscreen/screensizes)
+and [pixel densities](https://developer.android.com/training/multiscreen/screendensities).
+To ensure the Splash Screen looks good on all devices, you will need to generate
+the image for each pixel density.
+
+A full explanation of [display-independent pixels (dp or dip)](https://developer.android.com/training/multiscreen/screendensities#TaskUseDP) is beyond the scope of this article, but a good
```

Comment:
nit: wrap the line

Reply:
done

Reply:
![1 banner](https://user-images.githubusercontent.com/6325672/59135314-51b37980-89a0-11e9-8adf-bd01608828fd.png)
Digital Marketing! | Work From Home! 
Make money online without any expenses!
Our Journey Since 24.12.2012
Subscribe & help us to build a blog !
http://www.mizansurovi1.blogspot.com